.circleci/config.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     parallelism: 1
     docker:
       - image: "cypress/included:8.5.0"
-    working_directory: ~/NeetoUiChallenge
+    working_directory: ~/neetouichallenge
```

Comment:
This change doesn't belong in a PR that is titled `created note listing page UI`. A PR is not a dumping ground for all changes that is meant to be added to master branch. Each PR fixes an atomic issue.

For the change under review, you should have raised another issue and should've made a separate PR.